DAOS-Support.md

```diff
@@ -62,7 +65,7 @@ Show a copy from one DAOS container to another container that exists in the same
 pool. A DAOS Unified Namespace path is used as the source and the destination.
 
 ```shell
-$ mpirun -np 3 dcp -v /tmp/$USER/conts/p1cont1 /tmp/$USER/conts/p1cont2
+$ mpirun -np 3 dcp -v daos:///tmp/$USER/conts/p1cont1 daos:///tmp/$USER/conts/p1cont2
```

Comment:
@dsikich Similar to above, duns_resolve_path expects daos:// to provide the uuids, so the daos:// prefix should be omitted for UNS paths.